revenuecat_examples/MagicWeather/lib/src/views/home.dart

```diff
@@ -30,7 +30,7 @@ class AppContainerState extends State<AppContainer> {
 
   Future<void> initPlatformState() async {
     // Enable debug logs before calling `configure`.
-    await Purchases.setDebugLogsEnabled(true);
+    await Purchases.setLogLevel(LogLevel.DEBUG);
```

Comment:
I think this should be `LogLevel.debug`? Not sure why it's letting you set it to uppercase